client/cypress/seed-test-db.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+export FLASK_ENV=test
+trap 'kill 0' SIGINT SIGHUP
+pushd ..
```

Comment:
This wouldn't work for me unless I changed it to `../..`. It couldn't find the scripts modules because it was looking relative to the /client folder instead of /arlo.

Reply:
Ok I think i fixed this by making sure each script first `cd`s to the directory it lives in